legal-api/migrations/versions/ef8f033d317b_add_meta_data_to_batch_processing.py

```diff
@@ -0,0 +1,28 @@
+"""add_meta_data_to_batch_processing
+
+Revision ID: ef8f033d317b
+Revises: 508c7466b2af
+Create Date: 2024-05-16 16:24:16.922376
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+# revision identifiers, used by Alembic.
+revision = 'ef8f033d317b'
+down_revision = '508c7466b2af'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('batch_processing', sa.Column('meta_data', JSONB, nullable=True))
```

Comment:
May need some changes here to align with [others](https://github.com/bcgov/lear/blob/main/legal-api/migrations/versions/9c95a9b28eb3_adding_filing_meta_data.py#L20).

Reply:
Thank you for the feedback, @chenhongjing.
I will update the migration script accordingly.